cg/cli/upload/clinical_delivery.py

```diff
@@ -104,14 +105,24 @@ def auto_fastq(context: click.Context, dry_run: bool):
                 )
                 analysis_obj.uploaded_at = dt.datetime.now()
             else:
-                LOG.warning(
+                LOG.debug(
                     "Upload to clinical-delivery for %s has already started, skipping",
                     analysis_obj.family.internal_id,
                 )
             continue
         case: Family = analysis_obj.family
         LOG.info("Uploading family: %s", case.internal_id)
         analysis_obj.upload_started_at = dt.datetime.now()
-        context.invoke(upload_clinical_delivery, case_id=case.internal_id, dry_run=dry_run)
+        try:
+            context.invoke(upload_clinical_delivery, case_id=case.internal_id, dry_run=dry_run)
```

Comment:
Not a fan of this pattern where we call other CLI commands, very much out of scope though 😅

Reply:
Yep, it's the basis upon which cg is built 😢 If we can remove it in every module we re-write I'll be happy.